README.md

```diff
@@ -45,6 +45,7 @@ It's also the perfect companion for testing CLI applications built with commande
 - [Helpers](#helpers)
     - [**cast**](#cast-helper) [usage](#cast-usage) | [add a type](#add-a-type)
 - [Examples](#examples)   
+- [CLI](#cli)   
```

Comment:
IMO it should be just after the install and promoted as the standard way to use veggies

Reply:
Done.